src/url-replacements.js

```diff
@@ -324,8 +352,10 @@ class UrlReplacements {
   /**
    * Returns navigation timing information based on the start and end events.
    * The data for the timing events is retrieved from performance.timing API.
+   * If start and end events are both given, the result is the difference between the two.
+   * If only start event is given, the result is the timing value at start event.
    * @param {string} startEvent
-   * @param {string} endEvent
+   * @param {string} [endEvent]
```

Comment:
I think this repo uses the google closure style docs: http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values